h2o-automl/src/main/java/water/automl/api/schemas3/AutoMLV99.java

```diff
@@ -1,18 +1,29 @@
 package water.automl.api.schemas3;
 
 import ai.h2o.automl.AutoML;
+import ai.h2o.automl.EventLog;
 import ai.h2o.automl.Leaderboard;
-import ai.h2o.automl.UserFeedback;
 import water.DKV;
+import water.Iced;
+import water.Key;
 import water.api.API;
 import water.api.schemas3.KeyV3;
 import water.api.schemas3.SchemaV3;
 import water.api.schemas3.TwoDimTableV3;
 
 // TODO: this is about to change from SchemaV3 to RequestSchemaV3:
 public class AutoMLV99 extends SchemaV3<AutoML,AutoMLV99> {
+
+  public static class AutoMLKeyV3 extends KeyV3<Iced, AutoMLKeyV3, AutoML> {
+    public AutoMLKeyV3() { }
+
+    public AutoMLKeyV3(Key<AutoML> key) {
+      super(key);
+    }
+  }
+
```

Comment:
moved this schema class out of AutoML to group it with other schema definitions

Reply:
@seb-h2o  where was it before I wonder? :)  is it related to eventLog stuff?

Reply:
na, it was in AutoML